ietf-wdm-interface.yang

```diff
@@ -326,23 +227,24 @@ module ietf-ext-xponder-wdm-if {
     }
   }
 
-  notification opt-if-och-min-tca {
+  notification wdm-if-min-tca {
```

Comment:
Could you add a sentence to explain that TCA are stateless and then they are modeled as notifications, and not just reporting state data nodes changes?